app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="TextAppearance">
+        <item name="android:fontFamily">@font/bai_jamjuree_medium</item>
```

Comment:
For the base text appearance I prefer setting the regular version of the font  `bai_jamjuree_regular`. If you do this change, you'll need to set `bai_jamjuree_medium` to Body1